### PR TITLE
fix: replaced incorrect icons on wallet bottom sheet with correct icons

### PR DESCRIPTION
**Pull request by vinnyhoward**
## **Description**
Replaced incorrect icons with correct icons raised from this [issue](https://github.com/MetaMask/metamask-mobile/issues/8053) icon assets pulled from [Figma](https://www.figma.com/file/MlBUrNnIXlftX4TSjqiUYg/Core-Components?type=design&node-id=1213-7938&mode=design&t=jTWqDkNHlaMh3yiO-0)

There is a slight deviation from the design where `minus` was not shown in the Figma so I just increased the stroke of the current minus to be the same weight as the icons

Swapped out `SwapVertical` icon with compressed version and fixed typo related to `SwapVertical`

## **Related issues**

Fixes: N/A

## **Manual testing steps**

1. Click on action tab
2. Compare action tab from Figma to this build
3. Compare action tab from previous build to this build or just look at my screenshots

## **Screenshots/Recordings**

Bottom Sheet Changes
| Before  | After  |
|:---:|:---:|
|before_bottom_sheet|after_bottom_sheet|
 
Swap Vertical Icon Changes
| Before  | After  |
|:---:|:---:|
|before_swap_vertical|after_swap_vertical|

### **Before**
N/A

### **After**
N/A

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by vinnyhoward** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by mirceanis — DISMISSED** [review]
Looks good

**Review by EtherWizard33 — DISMISSED** [review]
LGTM

**Comment by EtherWizard33** [comment]
about the label check that is failing, i suspect it could be because of something seen last month, when someone added a job or action that automatically adds the team label provided based on the PR author name, it's possible that you will need to add your name to the repo or ask someone who has the rights to do so:

I believe that someone in this thread had the same issue https://consensys.slack.com/archives/C02U025CVU4/p1719431165050949

**Review by brianacnguyen — CHANGES_REQUESTED** [review]
Request change with new icon files that I sent

File: app/component-library/components/Icons/Icon/assets/swap-vertical.svg
Comment (brianacnguyen) [inline app/component-library/components/Icons/Icon/assets/swap-vertical.svg:1]: Can you update this svg with the ones i sent?

**Review by brianacnguyen — CHANGES_REQUESTED** [review]
Requesting SVG changes

File: app/components/Views/WalletActions/WalletActions.tsx [hunk 1]
```diff
@@ -134,7 +134,7 @@ const WalletActions = () => {
           <WalletAction
             actionTitle={strings('asset_overview.sell_button')}
             actionDescription={strings('asset_overview.sell_description')}
-            iconName={IconName.Minus}
+            iconName={IconName.MinusBold}
```
Comment (brianacnguyen) [inline app/components/Views/WalletActions/WalletActions.tsx:137]: Can you provide a screenshot/screencap for this change?

Comment (vinnyhoward) [reply app/components/Views/WalletActions/WalletActions.tsx:137]: The PR screenshots should be updated with them now

**Review by brianacnguyen — APPROVED** [review]

**Pull request merged**